src/app/content/components/BookBanner.tsx

```diff
@@ -139,7 +143,7 @@ export const BarWrapper = styled.div<{colorSchema: Book['theme'] | undefined , u
   position: ${ifMiniNav('sticky', 'relative' /* stay above mini nav */)};
   z-index: ${ifMiniNav(theme.zIndex.navbar - 2, theme.zIndex.navbar - 1)};
   overflow: hidden;
-  ${(props: {colorSchema: Book['theme'] | undefined }) => props.colorSchema && css`
+  ${(props: {colorSchema: BookWithOSWebData['theme'] | undefined }) => props.colorSchema && css`
```

Comment:
This is a good moment to refactor this to `colorSchema?: BookWithOSWebData['theme']`

Reply:
I'm not exactly sure if its possible for `colorSchema`  to be undefined now except if it was set explicitly to it which i don't think is meant to happen.

Reply:
If it can't then you can remove `| undefined`